src/editableui/editableuiview.js

```diff
@@ -39,7 +39,9 @@ export default class EditableUIView extends View {
 					'ck-content',
 					'ck-editor__editable',
 					'ck-rounded-corners',
-					bind.to( 'isFocused', value => value ? 'ck-focused' : 'ck-blurred' )
+					bind.to( 'isFocused', value => value ? 'ck-focused' : 'ck-blurred' ),
+					bind.to( 'isReadOnly', value => value ? 'ck-read-only' : '' )
```

Comment:
I think we can shorten this a little bit

```suggestion
					bind.if( 'isReadOnly', 'ck-read-only' )
```